src/css/components/_post-card.scss

```diff
@@ -0,0 +1,104 @@
+.post-card p {
```

Comment:
Do you reckon we'll have more than one type of card?  If so we should structure this and others into a cards folder inside components

Reply:
Actually just looking back into the components folder we already have a card  in there so I think the folder structure makes sense, could have one for scrolls as  well.  let me know if you think that's a good idea.

Reply:
I agree, already done!